build.sbt

```diff
@@ -53,12 +53,12 @@ lazy val root = project.in(file("."))
     resolvers += "Sonatype Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots/",
     organization := "com.safety-data",
     name := "akka-persistence-redis",
-    version := "0.4.0",
+    version := "0.5.0-SNAPSHOT",
```

Comment:
For such a small change, I think a patch release is better. That would then be `0.4.1`.
If tests are all passing, then you can directly publish the final patch release.